packages/devtools_app/lib/src/shared/theme.dart

```diff
@@ -388,6 +388,9 @@ extension DevToolsColorScheme on ColorScheme {
 
   Color get selectedRowColor =>
       isLight ? devtoolsBlue[600]! : devtoolsGrey[600]!;
+
+  Color get coverageHitColor => Colors.green[600]!;
+  Color get coverageMissColor => Colors.red[600]!;
```

Comment:
Can we generalize `treemapIncreaseColor` and `treemapDecreaseColor` from colors.dart and use those values here for consistency? IIRC we had some UXD help with selecting those colors so that they'd look good in light and dark mode. (though we probably still need some help making sure they are accessible, but at least if we use the same green / red we only have to update in one place)

Reply:
Sure, that works for me. I didn't see `red` or `green` elsewhere in this file, so I just picked something that looked reasonable.